core/src/main/java/org/hibernate/ogm/jpa/impl/OgmEntityManager.java

```diff
@@ -633,39 +616,4 @@ public CriteriaBuilder getCriteriaBuilder() {
 	public Metamodel getMetamodel() {
 		return hibernateEm.getMetamodel();
 	}
-
-	/**
```

Comment:
I've filed [OGM-865](https://hibernate.atlassian.net/browse/OGM-865) to track that issue. Don't think it's a huge problem.